dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import List
+
+from airflow_breeze.ci.build_params import BuildParams
+from airflow_breeze.utils.path_utils import get_airflow_sources_root
+from airflow_breeze.utils.run_utils import run_command
+
+NECESSARY_HOST_VOLUMES = [
```

Comment:
Also I have added host_volumes starting with `/`. Is it the right way to do it? @potiuk

Reply:
I think so - because you are prefixing it later with "${airflow_sources}" so it should be fine.